rollup.config.js

```diff
@@ -31,7 +33,7 @@ export default {
 				browser: true,
 				dedupe: ['svelte']
 			}),
-			commonjs(),
+			commonjs({ sourceMap: dev }),
```

Comment:
I'm not sure that disabling source maps for just this plugin in production makes sense. E.g. if you're using something like Sentry.io it's really helpful to have those available